app/views/returnFrequency/confirm_dates.scala.html

```diff
@@ -31,14 +31,20 @@
 
   @headings(messages("confirm_return_frequency.heading"))
 
-  <p id="p1"
-     class="form-group">
-      <strong class="bold">
-          @messages("confirm_return_frequency.newDates")
-      </strong>
-      @messages(s"return_frequency.${model.id}")
+
+  <p id="p1" class="form-group">
+        @messages("confirm_return_frequency.newDates")
```

Comment:
In the prototype there is a full stop after this sentence. As we don't want to just put the full stop in the view, you could update this message key to take in the new return frequency as a parameter like so:
```
confirm_return_frequency.newDates = Your new VAT Return dates will be {0}.
```
Then update this line to be something like:
```
@messages("confirm_return_frequency.newDates", s"return_frequency.${model.id}")
```

Reply:
updated